modules/simple_forms_api/app/controllers/simple_forms_api/v1/uploads_controller.rb

```diff
@@ -137,13 +137,18 @@ def submit_form_to_benefits_intake
           { form_number: params[:form_number], status:, uuid: confirmation_number }
         )
 
+        submission_archive_handler = SimpleFormsApi::S3Service::SubmissionArchiveHandler.new(
+          benefits_intake_uuid: confirmation_number
```

Comment:
@pennja I'm thinking we should only accept one `benefits_intake_uuid` at a time since this will, for now at least, only be used for one form submission at a time. What do you think?